ew/static/food.py

```diff
@@ -2049,6 +2049,17 @@
         acquisition = "smelting",
     ),
 ]
+EwFood(
```

Comment:
the EWFood object doesn't have a closing parenthesis and is below the square bracket on line 2051, when it should be above.